src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -51,10 +51,11 @@ const useStyles = makeStyles({
   },
   stickyCardContainer: {
     width: '100%',
-    position: 'sticky',
-    top: '100px',
+    position: 'static',
     [theme.breakpoints.up('md')]: {
       width: '50vw',
+      position: 'sticky',
```

Comment:
This `position: 'sticky'` is the same no matter the breakpoint, right? So it shouldn't be here, or it shouldn't be in the containing element

Reply:
I missed that you were keeping the "static" position on the small screen. That said, the default `position` value is already `static`, so do we have to call it out?